pkg/kubelet/kubelet.go

```diff
@@ -2176,6 +2209,10 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 			// TODO: Do we want to support this?
 			glog.Errorf("Kubelet does not support snapshot update")
 		}
+
+		// Add sourcesReady after the operation on pod finishes
```

Comment:
@saad-ali @jingxu97 Nice to have the comment, but I think what is important is to know why this is necessary? Why did we move it? And what happens if someone moves it back? What is the dependency?